src/components/Partenaires/APIDepot/api-depot-revision/APIDepotRevisionItem.tsx

```diff
@@ -24,8 +24,8 @@ export default function APIDepotRevisionItem({ id, codeCommune, validation, isCu
         )
       default:
         return (
-          <Tooltip title={validation && validation.errors.length > 0 ? 'Des erreurs de validations empèchent la publication, pour en savoir plus veuillez consulter le rapport de validation' : 'La révision est en attente de publication'}>
-            <Badge severity={validation && validation.errors.length > 0 ? 'error' : 'warning'}>Non publiée</Badge>
+          <Tooltip title={validation && validation.errors?.length > 0 ? 'Des erreurs de validations empèchent la publication, pour en savoir plus veuillez consulter le rapport de validation' : 'La révision est en attente de publication'}>
```

Comment:
Pour aller au bout de la logique, est-ce qu'on ne pourrait pas mettre ça ? (J'ai aussi ajouter une correction sur un accent.)
```suggestion
          <Tooltip title={validation?.errors?.length > 0 ? 'Des erreurs de validations empêchent la publication, pour en savoir plus veuillez consulter le rapport de validation' : 'La révision est en attente de publication'}>
```

Reply:
Done, c'est bizarre y'a une erreur de typescript avec ta suggestion. Pour que ça passe il faut donner une valeur de fallback. étrange qu'il n'y ait pas d'erreur avec le ? sur errors. Merci pour la faute d'orthographe :)